spec/requests/notifications_controller_spec.rb

```diff
@@ -245,6 +245,28 @@ def delete_notification(resp_code, matcher)
             ])
           end
 
+          it "doesn't include reviewables that are claimed by someone that's not the current user" do
+            SiteSetting.enable_experimental_sidebar_hamburger = true
```

Comment:
Just curious, is this only applicable for the sidebar or should we just exclude claimed reviewables everywhere?

Reply:
In the old notifications menu we didn't include reviewables at all so I need to enable the setting for the response of `/notifications` endpoint to include reviewables.